osl/interface_linux.go

```diff
@@ -17,15 +17,16 @@ import (
 type IfaceOption func(i *nwIface)
 
 type nwIface struct {
-	srcName     string
-	dstName     string
-	master      string
-	dstMaster   string
-	address     *net.IPNet
-	addressIPv6 *net.IPNet
-	routes      []*net.IPNet
-	bridge      bool
-	ns          *networkNamespace
+	srcName        string
+	dstName        string
+	master         string
+	dstMaster      string
+	address        *net.IPNet
+	addressIPv6    *net.IPNet
+	routes         []*net.IPNet
+	bridge         bool
+	ns             *networkNamespace
+	addressAliases []*net.IPNet
```

Comment:
I've not seen this term before- I went by the more "official" one (at least for linux):
https://en.wikipedia.org/wiki/IP_aliasing
Seems like secondary address is more commonly used on Cisco equipment.